core/server/pom.xml

```diff
@@ -249,6 +249,20 @@
           <!-- insert other configuration here (optional) -->
         </configuration>
       </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.19</version>
+          <configuration>
+            <argLine>-Djava.net.preferIPv4Stack=true -XX:MaxPermSize=512M</argLine>
+            <redirectTestOutputToFile>${test.output.redirect}</redirectTestOutputToFile>
+            <!-- Alphabetical order on even hours, reverse-alphabetical on odd hours  -->
+            <runOrder>hourly</runOrder>
+            <reuseForks>false</reuseForks>
+            <forkCount>2</forkCount> 
```

Comment:
Could we make this "1C" instead so that it uses one fork per core on the machine running the build?